pkg/alertmanager/statefulset.go

```diff
@@ -407,6 +407,16 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config) (*appsv1.S
 		})
 	}
 
+	var resources v1.ResourceRequirements = v1.ResourceRequirements{}
+	if config.ConfigReloaderCPU != "0" && config.ConfigReloaderMemory != "0" {
```

Comment:
What if we want no CPU limit but do want a memory limit? I expect this may cause some confusion if both have to be set in order to be applied.

Reply:
Agreed. Will change.

Reply:
Changed to set them individually. Please take a look :)